packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/vps-announcement-banner.constant.js

```diff
@@ -0,0 +1,7 @@
+export const TASKS_INFO = {
+  DEFAULT: 'http://travaux.ovh.net/?do=details&id=51831',
```

Comment:
No need to have link depending on subsidiaries but need link depending on region (Not sure it's available in the US)